setup_yarn.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
```

Comment:
You'll likely want to do `set -e` so that any unexpected failures do properly fail the script instead of silently continuing.